docs/install/install-knative-gcp.md

```diff
@@ -117,6 +117,7 @@
       |       CloudStorageSource        |                              roles/storage.admin                               |
       |      CloudSchedulerSource       |                           roles/cloudscheduler.admin                           |
       |      CloudAuditLogsSource       | roles/pubsub.admin, roles/logging.configWriter, roles/logging.privateLogViewer |
+      |      CloudBuildSource           |                              None(no roles needed for CloudBuildSource)        |
```

Comment:
```suggestion
      |      CloudBuildSource           |                              None (no roles needed for CloudBuildSource)        |
```

Although I'm surprised it doesn't need `roles/pubsub.subscriber`. What credential is eventually used to read the pubsub topic?

Reply:
Yep. roles/pubsub.subscriber is needed. I forgot to mention it because the init_script already granted the roles/pubsub.editor